grails-app/controllers/OauthAdminController.groovy

```diff
@@ -0,0 +1,123 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.annotation.Secured
+import org.springframework.security.oauth2.provider.token.TokenStore;
+import org.transmart.oauth2.Client
+
+
+@Secured(['ROLE_ADMIN'])
```

Comment:
This is not enabled in transmart. We have `securityConfigType = 'InterceptUrlMap'`.